runtime/appruntime/exported/stack/stack.go

```diff
@@ -16,13 +16,40 @@ type Stack struct {
 }
 
 func Build(skip int) Stack {
+	return BuildWithFilters(skip+1, nil)
+}
+
+func BuildWithoutGoRuntime(skip int) Stack {
```

Comment:
Maybe make this a built-in filter function instead? `func ExcludeRuntime(pc uintptr) bool` or whatever, with usage `stack.BuildWithFilters(1, stack.ExcludeRuntime)`?